layers/meta-resin-generic/recipes-support/resin-init/resin-init-flasher-board/resin-init-flasher-board

```diff
@@ -1,7 +1,33 @@
 #!/bin/sh
 
+INSTALL_UEFI=""
+INTERNAL_DEVICE_BOOT_PART_MOUNTPOINT=/tmp/internal_boot
+
+efibootmgr &>/dev/null ; INSTALL_UEFI=$?
+
 set -e
 
+# Mount internal device boot partition
+mkdir -p $INTERNAL_DEVICE_BOOT_PART_MOUNTPOINT
+echo "[resin-init-flasher-board] Mounting internal device boot partition."
+if ! mount /dev/disk/by-label/resin-boot $INTERNAL_DEVICE_BOOT_PART_MOUNTPOINT; then
+    echo "[resin-init-flasher-board] Unable to mount boot partition"
+    exit 1
+fi
+
+if [ $INSTALL_UEFI -ne 0 ]; then
+    echo "[resin-init-flasher-board] non-UEFI installation."
+    rm -rf "$INTERNAL_DEVICE_BOOT_PART_MOUNTPOINT/EFI"
```

Comment:
I'm not sure what this does and why?

Reply:
If we choose an non-UEFI installation we remove the UEFI files.

Reply:
Sure. But why?